### ISSUE TITLE
Problem with config.py file

### ISSUE THREAD

**Issue by mzkhan141** [llSourcell/AI_Writer#5]
It looks like generate.py and config.py files are the same . I dont know if this is purposely same or most probably by mistake because there is no attribute `FLAG_CPU_MODE` and `paths` in config.py

**Comment by shivaraj-bh** [comment]
Yeah you are right, both the files are same which is not supposed to  be the case. You can actually replace that config.py with the one here, https://github.com/ryankiros  and got to his neural_storyteller repository.

**Comment by amanullahtariq** [comment]
You can use config file of the [ryankiros](https://github.com/ryankiros)
